client/src/components/TodoListItem.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import '../App.css';
+import {graphql} from 'babel-plugin-relay/macro';
+import { useFragment } from 'react-relay/hooks';
+import type {TodoListItem_todo$key} from './__generated__/TodoListItem_todo.graphql';
+import Checkbox from '@button-inc/component-library/Checkbox'
+
+type Props = {
+    //$key includes data and fragment reference
+    todo: TodoListItem_todo$key
+}
+
+function TodoListItem(props: Props) {
+    const data = useFragment(
+        graphql`
+          fragment TodoListItem_todo on Todo {
+            id
+            task
+            completed
+          }
+        `,
+        props.todo,
+      );
```

Comment:
When working with queries and components, these are the to-dos?
-write fragment in component
-spread fragment in parent

What order? Types etc. don't exist until you run the compiler, so there are always red lines and errors